shell/packages/sandstorm-ui-powerbox/powerbox-server.js

```diff
@@ -200,8 +263,12 @@ specialCaseTypes[Grain.UiView.typeId] = function (db, userId, value) {
 
   // TODO(someday): Allow `value` to specify app IDs to filter for.
 
-  const sharedGrainIds = db.userApiTokens(userId).map(token => token.grainId);
-  const ownedGrainIds = Grains.find({ userId: userId }, { fields: { _id: 1 } }).map(grain => grain._id);
+  const sharedGrainIds = db.userApiTokens(userId).fetch()
```

Comment:
Should userApiTokens() and userGrains() maybe be updated to ignore trash by default, or given a parameter to control the query?